Components/Alert/alert.css

```diff
@@ -0,0 +1,53 @@
+.alert{
+    width: 100%;
+    color:white;
```

Comment:
You can use a variable for `color: white` since its being repeated multiple times.

Reply:
change has been made